root/static/scripts/tests/guessFeat.js

```diff
@@ -8,7 +8,7 @@ var _ = require('lodash');
 var guessFeat = require('../edit/utility/guessFeat');
 
 test('guessing feat. artists', function (t) {
-    t.plan(18);
+    t.plan(17);
```

Comment:
This was just wrong...not sure why it didn't cause problems in phantomjs, but it did under node